src/routes/SelectRepos/index.js

```diff
@@ -82,6 +82,20 @@ export class SelectRepos extends React.Component {
           Can&#8217;t find one of your repos here? Make sure you are watching it
           on Github!
         </p>
+        {this.props.selectedRepos.length ? (
+          <p className={theme.intro}>
+            <Button
+              className={theme.button}
+              onClick={() => {
+                this.props.resetRepoSelection();
+              }}
+            >
+              Reset repo selection
+            </Button>
```

Comment:
The conditional rendering of the button needs adding to the component tests.

Reply:
😕